fvm/src/blockstore/buffered.rs

```diff
@@ -171,31 +172,59 @@ fn copy_rec<'a>(
     root: Cid,
     buffer: &mut Vec<(Cid, &'a [u8])>,
 ) -> Result<()> {
-    // TODO: Make this non-recursive.
-    // Skip identity and Filecoin commitment Cids
-    if root.codec() != DAG_CBOR {
-        return Ok(());
-    }
-
-    let block = &*cache
-        .get(&root)
-        .ok_or_else(|| anyhow!("Invalid link ({}) in flushing buffered store", root))?;
-
-    scan_for_links(&mut Cursor::new(block), |link| {
-        if link.codec() != DAG_CBOR {
-            return Ok(());
+    const DAG_RAW: u64 = 0x55;
+    const BLAKE2B_256: u64 = 0xb220;
+    const BLAKE2B_LEN: u8 = 32;
+    const IDENTITY: u64 = 0x0;
+
+    // Differences from lotus (vm.Copy):
+    // 1. We assume that if we don't have a blcok in our buffer, it must already be in lotus. We
```

Comment:
```suggestion
    // 1. We assume that if we don't have a block in our buffer, it must already be in the client.
```

Can we rewrite this to not be Lotus-specific? Lotus is not the only client integrating the FVM.